### ISSUE TITLE
Yoast Breadcrumb Typography issue

### ISSUE THREAD

**Issue by JennyPower** [elementor/elementor#21932]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The typography setting on the Yoast Breadcrumbs block does not apply styling to the links.

### Steps to reproduce

1) Add the Yoast Breadcrumbs block to the page
2) Set the typography to 12px font size

The links in the breadcrumb will not be 12px (non-linked crumbs are fine) and there's no way that I can see of changing the size except to use custom CSS

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.12
	PHP Version: 8.0.28
	PHP Memory Limit: 256M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://webtree.website/renom
	Home URL: https://webtree.website/renom
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-AU
	Timezone: 10
	Admin Email: jenny@webtree.com.au
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_AU
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

== Active Plugins ==
	Duplicator Pro
		Version: 4.5.10
		Author: Snap Creek

	Elementor
		Version: 3.12.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.1
		Author: Elementor.com

	ManageWP - Worker
		Version: 4.9.16
		Author: GoDaddy

	Unlimited Elements for Elementor (Premium)
		Version: 1.5.58
		Author: Unlimited Elements

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence

	WP-Optimize - Clean, Compress, Cache
		Version: 3.2.14
		Author: David Anderson, Ruhani Rabin, Team Updraft

	Yoast SEO
		Version: 20.4
		Author: Team Yoast

== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 
		Author: GoDaddy

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Editor Top Bar: Inactive
	Landing Pages: Inactive
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active
	Notes: Inactive
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Inactive
	Menu: Inactive by default

== Log ==
	
Log: showing 20 of 352023-03-20 13:37:51 [info] elementor::elementor_updater Started 
2023-03-20 13:37:51 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-20 13:37:51 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-20 13:37:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.1',
  'to' => '3.11.5',
)]
2023-03-20 13:37:52 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.1',
  'to' => '3.11.6',
)]
2023-04-03 13:04:57 [info] elementor-pro::elementor_pro_updater Started 
2023-04-03 13:04:57 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-03 13:04:57 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-03 13:04:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.11.7',
)]
2023-04-03 15:45:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-03 15:45:15 [info] elementor-pro::elementor_pro_updater Started 
2023-04-03 15:45:15 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-03 15:45:15 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-03 15:45:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-03 15:45:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-03 15:45:15 [info] elementor::elementor_updater Started 
2023-04-03 15:45:15 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-03 15:45:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-03 15:45:16 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-03 15:45:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]

JS: showing 2 of 2JS: 2023-03-20 05:42:24 [error X 1][https://webtree.website/renom/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.5:3:791161] Cannot read properties of undefined (reading 'toLowerCase') 
JS: 2023-04-04 05:59:42 [error X 1][https://webtree.website/renom/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.1:3:632658] Cannot read properties of undefined (reading 'isDesignable') 

PHP: showing 5 of 5PHP: 2023-03-20 16:04:48 [warning X 36][/home/webtreew/public_html/renom/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-21 08:26:19 [warning X 2][/home/webtreew/public_html/renom/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-21 08:59:46 [error X 4][/home/webtreew/public_html/renom/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on null in /home/webtreew/public_html/renom/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /home/webtreew/public_html/renom/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /home/webtreew/public_html/renom/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 /home/webtreew/public_html/renom/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /home/webtreew/public_html/renom/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#5 /home/webtreew/public_html/renom/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /home/webtreew/public_html/renom/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-21 09:21:42 [error X 6][/home/webtreew/public_html/renom/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /home/webtreew/public_html/renom/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /home/webtreew/public_html/renom/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /home/webtreew/public_html/renom/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 /home/webtreew/public_html/renom/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /home/webtreew/public_html/renom/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#5 /home/webtreew/public_html/renom/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /home/webtreew/public_html/renom/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-21 11:15:57 [warning X 17][/home/webtreew/public_html/renom/wp-content/plugins/elementor/app/modules/import-export/module.php::629] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Unlimited Elements for Elementor (Premium): Compatibility not specified

== Elementor Pro - Compatibility Tag ==
